Feat/linted

daveryan23 opened the pull request:
#6 lib/index.js has been linted. npm run build now works properly.

#7 .npmignore now contains 'test'. (Tests can't be run anyway without the 'lib' directory)

After merging this feature:
- Please delete the feature branch
- Publish new version on npm (v0.2.1) after running the build locally.

Thanks!

daveryan23 commented:
Yes please! Can you make me a collaborator on npm too? :) Look forward to functional v0.2.1. Sorry for the inconvenience. David


________________________________
From: Sebastian Duque Restrepo <notifications@github.com>
Sent: 05 July 2018 02:21
To: sebas095/primes-and-factors
Cc: David Ryan; Author
Subject: Re: [sebas095/primes-and-factors] Feat/linted (#8)

ok, in a moment I will organize it


--
Sebastian Duque Restrepo
Ingeniero de Sistemas y Computación





________________________________
De: David Ryan <notifications@github.com>
Enviado: miércoles, 27 de junio de 2018 1:52 a.m.
Para: sebas095/primes-and-factors
Cc: Subscribed
Asunto: [sebas095/primes-and-factors] Feat/linted (#8)


#6<https://github.com/sebas095/primes-and-factors/issues/6> lib/index.js has been linted. npm run build now works properly.

#7<https://github.com/sebas095/primes-and-factors/issues/7> .npmignore now contains 'test'. (Tests can't be run anyway without the 'lib' directory)

After merging this feature:

* Please delete the feature branch
* Publish new version on npm (v0.2.1) after running the build locally.

Thanks!

________________________________
You can view, comment on, or merge this pull request online at:

https://github.com/sebas095/primes-and-factors/pull/8

Commit Summary

* index.js passes lint. Tests at 100%.
* test dir added to .npmignore

File Changes

* M .npmignore<https://github.com/sebas095/primes-and-factors/pull/8/files#diff-0> (1)
* M lib/index.js<https://github.com/sebas095/primes-and-factors/pull/8/files#diff-1> (16)

Patch Links:

* https://github.com/sebas095/primes-and-factors/pull/8.patch
* https://github.com/sebas095/primes-and-factors/pull/8.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/sebas095/primes-and-factors/pull/8>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AIc_D_kW15Q7v0EpmzlFnRi5Hn0O1tGsks5uAyvJgaJpZM4U5Jfj>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsebas095%2Fprimes-and-factors%2Fpull%2F8%23issuecomment-402546432&data=02%7C01%7C%7C5f5b33a0f19241e2971108d5e1e3696a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636663289194571014&sdata=vgyivc%2FNPjXCevsxA9XIgEViJjE5rACkXcl2kJ6OiO8%3D&reserved=0>, or mute the thread<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FALVSJuCBg7SDCpryHXVQaSBzUMHlWfKgks5uDRXWgaJpZM4U5Jfj&data=02%7C01%7C%7C5f5b33a0f19241e2971108d5e1e3696a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636663289194571014&sdata=0%2FLiws66r9O%2Byiv1eJyM0V70OMVjyOAYTfBeVrH0LLY%3D&reserved=0>.

sebas095 commented:
I already do the merge and also I added you as collaborator in npm


--

Sebastian Duque Restrepo
Ingeniero de Sistemas y Computación





________________________________
De: David Ryan <notifications@github.com>
Enviado: domingo, 8 de julio de 2018 5:18 p.m.
Para: sebas095/primes-and-factors
Cc: Sebastian Duque Restrepo; Comment
Asunto: Re: [sebas095/primes-and-factors] Feat/linted (#8)

Yes please! Can you make me a collaborator on npm too? :) Look forward to functional v0.2.1. Sorry for the inconvenience. David


________________________________
From: Sebastian Duque Restrepo <notifications@github.com>
Sent: 05 July 2018 02:21
To: sebas095/primes-and-factors
Cc: David Ryan; Author
Subject: Re: [sebas095/primes-and-factors] Feat/linted (#8)

ok, in a moment I will organize it


--
Sebastian Duque Restrepo
Ingeniero de Sistemas y Computación





________________________________
De: David Ryan <notifications@github.com>
Enviado: miércoles, 27 de junio de 2018 1:52 a.m.
Para: sebas095/primes-and-factors
Cc: Subscribed
Asunto: [sebas095/primes-and-factors] Feat/linted (#8)


#6<https://github.com/sebas095/primes-and-factors/issues/6> lib/index.js has been linted. npm run build now works properly.

#7<https://github.com/sebas095/primes-and-factors/issues/7> .npmignore now contains 'test'. (Tests can't be run anyway without the 'lib' directory)

After merging this feature:

* Please delete the feature branch
* Publish new version on npm (v0.2.1) after running the build locally.

Thanks!

________________________________
You can view, comment on, or merge this pull request online at:

https://github.com/sebas095/primes-and-factors/pull/8

Commit Summary

* index.js passes lint. Tests at 100%.
* test dir added to .npmignore

File Changes

* M .npmignore<https://github.com/sebas095/primes-and-factors/pull/8/files#diff-0> (1)
* M lib/index.js<https://github.com/sebas095/primes-and-factors/pull/8/files#diff-1> (16)

Patch Links:

* https://github.com/sebas095/primes-and-factors/pull/8.patch
* https://github.com/sebas095/primes-and-factors/pull/8.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/sebas095/primes-and-factors/pull/8>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AIc_D_kW15Q7v0EpmzlFnRi5Hn0O1tGsks5uAyvJgaJpZM4U5Jfj>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsebas095%2Fprimes-and-factors%2Fpull%2F8%23issuecomment-402546432&data=02%7C01%7C%7C5f5b33a0f19241e2971108d5e1e3696a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636663289194571014&sdata=vgyivc%2FNPjXCevsxA9XIgEViJjE5rACkXcl2kJ6OiO8%3D&reserved=0>, or mute the thread<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FALVSJuCBg7SDCpryHXVQaSBzUMHlWfKgks5uDRXWgaJpZM4U5Jfj&data=02%7C01%7C%7C5f5b33a0f19241e2971108d5e1e3696a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636663289194571014&sdata=0%2FLiws66r9O%2Byiv1eJyM0V70OMVjyOAYTfBeVrH0LLY%3D&reserved=0>.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/sebas095/primes-and-factors/pull/8#issuecomment-403321633>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AIc_D3o2tViheOJKzrG2a9vuf_k_RPxLks5uEoUngaJpZM4U5Jfj>.

daveryan23 commented:
Thanks Sebastian!


I have now:

- Pulled your changes locally, deleted old branches locally

- Updated version number to 0.2.0 manually

- Used 'npm version patch' to update version number to 0.2.1 and tag this version

- Used 'git push' to update GitHub

- Used 'npm publish' to publish to npm

- Tested installing the package 0.2.1 in another local project - it appeared to work.


So I guess that's all good then!


Interestingly, I'm listed as an 'administrator' on npm, not a 'collaborator'. I'm not sure what the difference is.


Kind regards,

David Ryan




________________________________
From: Sebastian Duque Restrepo <notifications@github.com>
Sent: 09 July 2018 21:33
To: sebas095/primes-and-factors
Cc: David Ryan; Author
Subject: Re: [sebas095/primes-and-factors] Feat/linted (#8)

I already do the merge and also I added you as collaborator in npm


--

Sebastian Duque Restrepo
Ingeniero de Sistemas y Computación





________________________________
De: David Ryan <notifications@github.com>
Enviado: domingo, 8 de julio de 2018 5:18 p.m.
Para: sebas095/primes-and-factors
Cc: Sebastian Duque Restrepo; Comment
Asunto: Re: [sebas095/primes-and-factors] Feat/linted (#8)

Yes please! Can you make me a collaborator on npm too? :) Look forward to functional v0.2.1. Sorry for the inconvenience. David


________________________________
From: Sebastian Duque Restrepo <notifications@github.com>
Sent: 05 July 2018 02:21
To: sebas095/primes-and-factors
Cc: David Ryan; Author
Subject: Re: [sebas095/primes-and-factors] Feat/linted (#8)

ok, in a moment I will organize it


--
Sebastian Duque Restrepo
Ingeniero de Sistemas y Computación





________________________________
De: David Ryan <notifications@github.com>
Enviado: miércoles, 27 de junio de 2018 1:52 a.m.
Para: sebas095/primes-and-factors
Cc: Subscribed
Asunto: [sebas095/primes-and-factors] Feat/linted (#8)


#6<https://github.com/sebas095/primes-and-factors/issues/6> lib/index.js has been linted. npm run build now works properly.

#7<https://github.com/sebas095/primes-and-factors/issues/7> .npmignore now contains 'test'. (Tests can't be run anyway without the 'lib' directory)

After merging this feature:

* Please delete the feature branch
* Publish new version on npm (v0.2.1) after running the build locally.

Thanks!

________________________________
You can view, comment on, or merge this pull request online at:

https://github.com/sebas095/primes-and-factors/pull/8

Commit Summary

* index.js passes lint. Tests at 100%.
* test dir added to .npmignore

File Changes

* M .npmignore<https://github.com/sebas095/primes-and-factors/pull/8/files#diff-0> (1)
* M lib/index.js<https://github.com/sebas095/primes-and-factors/pull/8/files#diff-1> (16)

Patch Links:

* https://github.com/sebas095/primes-and-factors/pull/8.patch
* https://github.com/sebas095/primes-and-factors/pull/8.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/sebas095/primes-and-factors/pull/8>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AIc_D_kW15Q7v0EpmzlFnRi5Hn0O1tGsks5uAyvJgaJpZM4U5Jfj>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsebas095%2Fprimes-and-factors%2Fpull%2F8%23issuecomment-402546432&data=02%7C01%7C%7C5f5b33a0f19241e2971108d5e1e3696a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636663289194571014&sdata=vgyivc%2FNPjXCevsxA9XIgEViJjE5rACkXcl2kJ6OiO8%3D&reserved=0>, or mute the thread<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FALVSJuCBg7SDCpryHXVQaSBzUMHlWfKgks5uDRXWgaJpZM4U5Jfj&data=02%7C01%7C%7C5f5b33a0f19241e2971108d5e1e3696a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636663289194571014&sdata=0%2FLiws66r9O%2Byiv1eJyM0V70OMVjyOAYTfBeVrH0LLY%3D&reserved=0>.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/sebas095/primes-and-factors/pull/8#issuecomment-403321633>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AIc_D3o2tViheOJKzrG2a9vuf_k_RPxLks5uEoUngaJpZM4U5Jfj>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsebas095%2Fprimes-and-factors%2Fpull%2F8%23issuecomment-403500387&data=02%7C01%7C%7C0e26c92c9d8143adfd5608d5e5a90127%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636667436383035178&sdata=PAyH4NjQYnV1PK%2Ff4SE5sTDpjMfmT0VXk6l6FUrw3Pc%3D&reserved=0>, or mute the thread<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FALVSJh8AJwyOlg4_Ln3oJVtlz957WZKCks5uE2nUgaJpZM4U5Jfj&data=02%7C01%7C%7C0e26c92c9d8143adfd5608d5e5a90127%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636667436383035178&sdata=e4zLOn784jMCru6f5jp4opf4jeLGFn0RpEd6sJIinok%3D&reserved=0>.
